pkgs/development/python-modules/download/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  wheel,
+  requests,
+  six,
+  tqdm,
+}:
+
+buildPythonPackage rec {
+  pname = "download";
+  version = "0.3.5";
+  pyproject = true;
+
+  src = fetchPypi {
```

Comment:
Fetching from GitHub is preferred unless there is a compelling reason not to

Reply:
Like I wrote for another, I am not sure where this recommendation is coming from? I haven't been able to find it in the docs (which provide several examples using `fetchPypi`). There are also 6k+ packages using `fetchPypi` in nixpkgs right now.